### PR TITLE
Redistribute olx types for ol/interaction

### PR DESCRIPTION
**Pull request by ahocevar**
See #7947.

### CODE REVIEW

File: src/ol/interaction/DragBox.js [hunk 1]
```diff
@@ -75,18 +97,18 @@ inherits(DragBoxEvent, Event);
...
+ * (see {@link module:ol/interaction/DragZoom~DragZoom} and
+ * {@link module:ol/interaction/DragRotateAndZoom}).
  *
  * This interaction is only supported for mouse devices.
  *
  * @constructor
- * @extends {ol.interaction.Pointer}
- * @fires ol.interaction.DragBoxEvent
- * @param {olx.interaction.DragBoxOptions=} opt_options Options.
+ * @extends {module:ol/interaction/Pointer}
```
Comment (tschaub) [inline src/ol/interaction/DragBox.js]: Should this be Pointer~Pointer?

File: src/ol/interaction/DragRotateAndZoom.js [hunk 2]
```diff
@@ -8,6 +8,17 @@ import {shiftKeyOnly, mouseOnly} from '../events/condition.js';
...
 import PointerInteraction from '../interaction/Pointer.js';
 
+
+/**
+ * @typedef {Object} Options
+ * @property {module:ol/events/condition~Condition} [condition] A function that
+ * takes an {@link module:ol/MapBrowserEvent~MapBrowserEvent} and returns a
+ * boolean to indicate whether that event should be handled.
+ * Default is {@link module:ol/events/condition~shiftKeyOnly}.
+ * @property {number} [dura00tion=4] Animation duration in milliseconds.
```
Comment (tschaub) [inline src/ol/interaction/DragRotateAndZoom.js]: Looks like some extra 00.

Comment (tschaub) [reply src/ol/interaction/DragRotateAndZoom.js]: Or duration=400?

**Comment by ahocevar** [comment]
Thanks for the sharp eye @tschaub. I've fixed those.

**Comment by ahocevar** [comment]
Thanks for the review @tschaub.

**Pull request merged**